com.ibm.streamsx.jdbc/impl/java/src/com/ibm/streamsx/jdbc/AbstractJDBCOperator.java

```diff
@@ -482,28 +482,30 @@ public synchronized void shutdown() throws Exception {
 
     }
   
-    // Set up JDBC driver class path
+	// Set up JDBC driver class path
 	private void setupClassPath(OperatorContext context) throws MalformedURLException{	
-        	// Split the jdbcDriverLib by "/"	
-		StringTokenizer st = new StringTokenizer(jdbcDriverLib , File.separator);
-		String libDir = st.nextToken();
+
+		String libDir = jdbcDriverLib.substring(0, jdbcDriverLib.lastIndexOf(File.separator));

```

Comment:
This line may fail if the jdbcDriverLib string does not contain any file separator character, not sure if this can happen ...